surround/tests/cli/remote_cli/add_test.py

```diff
@@ -1,40 +0,0 @@
-import os
-import shutil
-import unittest
-import subprocess
-
-__author__ = 'Akshat Bajaj'
-__date__ = '2019/03/04'
-
-class AddTest(unittest.TestCase):
```

Comment:
Instead of removing the tests altogether, probably best to update them with the new add....